src/domains/User/UserSettings/UserSettings.tsx

```diff
@@ -144,6 +145,17 @@ export default function UserSettings({ id }: Props) {
             component: <NotificationSettings user={user} />,
         },
         // {
+        //     title: 'GeneralEditableText',
+        //     description: 'General, but with EditableText instead of TextField',
+        //     component: (
+        //         <Grid container spacing={2}>
+        //             <Grid item xs={12}>
+        //                 <UserProfileEditable img='https://i.imgur.com/3beQH5s.jpeg' />
+        //             </Grid>
+        //         </Grid>
+        //     ),
+        // },
+        // {
```

Comment:
Is this section still gonna be used?  Should probably erase it if it's gonna be commit while commented out (unless there's a reason I am unaware of).

Reply:
Done
(BTW, I did keep one commented out section in `UserProfile.test.tsx` since I am still working on it and do not want it cluttering the test results when running `yarn test`)